app/src/componentLibrary/attributeEditor/attributeEditor.jsx

```diff
@@ -131,14 +142,19 @@ export const AttributeEditor = ({
           onInputChange={handleAttributeKeyInputChange}
           autocompleteVariant={'key-variant'}
         />
-      </div>
+      </FieldErrorHint>
       <div className={cx('separator')}>:</div>
-      <div>
+      <FieldErrorHint
+        provideHint={false}
+        error={state.errors.value}
+        touched={valueTouched}
+        setTouch={setTouchValue}
+      >
         <AttributeInput
           minLength={1}
           attributes={attributes}
           attributeComparator={byValueComparator}
-          getURI={getURIValue(state.key)}
+          getURI={getURIValue}
```

Comment:
Is it not needed anymore?
Previously attribute value search was done using `key` to find necessary values from pairs, f.e.:
```javascript
key: 'build',
value: '1.7.2' | '1.6.4' | '1.7.3' etc...
```